src/utils/daos_hdlr.c

```diff
@@ -1568,6 +1572,469 @@ cont_destroy_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+static int
+copy_recx_single(daos_key_t *dkey,
+		 daos_handle_t *src_oh,
+		 daos_handle_t *dst_oh,
+		 daos_iod_t *iod)
+{
+	/* if iod_type is single value just fetch iod size from source
+	 * and update in destination object */
+	int         buf_len = (int)(*iod).iod_size;
+	char        buf[buf_len];
+	d_sg_list_t sgl;
+	d_iov_t     iov;
+	int	    rc;
+
+	/* set sgl values */
+	sgl.sg_nr     = 1;
+	sgl.sg_nr_out = 0;
+	sgl.sg_iovs   = &iov;
+	d_iov_set(&iov, buf, buf_len);
+        rc = daos_obj_fetch(*src_oh, DAOS_TX_NONE, 0, dkey, 1, iod, &sgl, NULL, NULL);
```

Comment:
(style)  line over 80 characters
(style)  code indent should use tabs where possible